### ISSUE TITLE
AttributeError: '_thread._local' object has no attribute 'srcdc'

### ISSUE THREAD

**Issue by Mizkau** [fishyboteso/fishyboteso#176]
Hi, 

I have a issue : 

```
INFO - Starting the bot engine, look at the fishing hole to start fishing
ERROR - Traceback (most recent call last):
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\fishy\engine\common\window_server.py", line 84, in run
    loop()
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\fishy\engine\common\window_server.py", line 71, in loop
    WindowServer.Screen = get_cropped_screenshot()
                          ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\fishy\engine\common\window_server.py", line 49, in get_cropped_screenshot
    ss = WindowServer.sslib.grab()
         ^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\fishy\engine\common\screenshot.py", line 51, in grab
    sct_img = self.sct.grab(self.sct.monitors[self.monitor_id])
              ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\mss\base.py", line 90, in grab
    screenshot = self._grab_impl(monitor)
                 ^^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\myuser\AppData\Local\Programs\Python\Python311\Lib\site-packages\mss\windows.py", line 235, in _grab_impl
    srcdc, memdc = self._handles.srcdc, self._handles.memdc
                   ^^^^^^^^^^^^^^^^^^^
AttributeError: '_thread._local' object has no attribute 'srcdc'
```

Did i install it wrong ?

**Comment by adsau59** [comment]
u need to change ur screenshot library

**Comment by orcaofdoom** [comment]
What do you mean change screenshot library from debug to screenshot lib? Like change my screenshot save location?

**Comment by adsau59** [comment]
will be fixed through #173

**Comment by adsau59** [comment]
closing issue, as now screenshot lib should switch automatically when not working
